src/scportrait/plotting/vis.py

```diff
@@ -5,19 +5,28 @@
 import numpy as np
 from skimage.color import label2rgb
 
+from scportrait.pipeline.project import Project
```

Comment:
This might be the reason for the circular import. I don't have time to look at it atm. Maybe at a later point

Reply:
I ran into a similar problem elsewhere when I tried to add Project as a type resulting in a circular import. I unfortunately didn't find a fix and then did not do explicit typing for the Project class.